### PR TITLE
Update Password Policy

### PR DESCRIPTION
**Pull request by EThom73**
An update to the password policy to include:
- Our new rules for password generation. 
- Guidance for the team around social engineering as a way of passwords being stolen
- confirming we will try to enable 2FA wherever possible

### CODE REVIEW

File: guides/security/password_policy.md [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
-- [ ] Use a secure password generator to generate complex passwords
-- [ ] Password length should be a minimum of 8 characters
-- [ ] Length is crucial because it increases entropy (number of uncertainty)
-- [ ] Never write down or share your passwords
-- [ ] Do not use a password for more than one account (reused passwords should be flagged by LastPass)
-- [ ] Password manager should follow this password policy
+## Password Management
+We use password management for Made Tech services. Individual vaults are managed by the teams who use them, and the password manager is managed by the Operations Team.
 
+Your password to access your password manager should confirm to the password policy. Biometrics is also an acceptable method of accessing the password manager.
```
Comment (chrisblackburn) [inline guides/security/password_policy.md]: conform

File: guides/security/password_policy.md [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+- Do not reveal a password on forms
+- Do not hint at the format of a password ("my hometown/first pet's name/etc was...")
+- Be sceptical of unusual requests
+- Be aware of potential phishing attacks
+- Consider who may be shoulder surfing
+- Hover your mouse over links in e-mail to ensure the web address makes sense
+- Verify the identity of a requester before sharing information
+- When in doubt, do not respond!
+
+It's always better to be safe than sorry, so report anything you think is amiss!
```
Comment (chrisblackburn) [inline guides/security/password_policy.md]: more safe than sorry

File: guides/security/password_policy.md [hunk 3]
```diff
@@ -1,12 +1,38 @@
...
-- [ ] Do not use a password for more than one account (reused passwords should be flagged by LastPass)
-- [ ] Password manager should follow this password policy
+## Password Management
+We use password management for Made Tech services. Individual vaults are managed by the teams who use them, and the password manager is managed by the Operations Team.
 
+Your password to access your password manager should confirm to the password policy. Biometrics is also an acceptable method of accessing the password manager.
+
+## Password Security
+
+### Obtaining Passwords via Social Engineering
```
Comment (chrisblackburn) [inline guides/security/password_policy.md]: This feels like the least important part of this guidance, yet it takes up the most space.

What's really important?

- Be aware of someone using social engineering to gain access to your password / account
- Do not store passwords on your device outside of an encrypted password manager
- Do not write passwords down
- Be sure you're entering your password in to a legitimate website
- Passwords should generally not be shared. When these is need to share a piece of secure information such as a password do so via 1Password

File: guides/security/password_policy.md (hunk 1 above)
Comment (EThom73) [reply guides/security/password_policy.md]: corrected 👍

File: guides/security/password_policy.md (hunk 2 above)
Comment (EThom73) [reply guides/security/password_policy.md]: removed

File: guides/security/password_policy.md (hunk 3 above)
Comment (EThom73) [reply guides/security/password_policy.md]: deleted this section and put the above points in

**Pull request merged**